### DISCUSSION TITLE
firefox beta : trim reblogs error

### DISCUSSION THREAD

**Discussion by sylastcr** [AprilSylph/XKit-Rewritten#1577] [Help Desk]
i waa hoping someone out here would be able to help me because i have tried everything and cannot seem to find a solution.

i reblog a post, add my own text to it, post it. then go on to trim it and get the error seen in the picture every single time on different posts.
![Screenshot_20240818_232241_com_kiwibrowser_browser_ChromeTabbedActivity](https://github.com/user-attachments/assets/9983130f-b14f-442e-8c8c-ea08786961c0)

if anyone has any idea of how to fix it, please let me know. i have uninstalled firefox, xkit, tried it on my laptop on chrome using xkit rewritten, nothing works.

**Comment by marcustyphoon** [comment]
Ooh, interesting. I can't tell if this is a Tumblr server thing or us. Thanks for trying different posts and different browsers, that's helpful info and removes a lot of the possibilities. There's very little code being run here and I can see from your screenshot that the post in question is pretty normal.

Is there any media (like images) in the content you're adding to the post before you try to trim it? Are there any tags with fancy characters? If there are either, can you try doing the same thing without the images/without the tags? If we can find a specific thing that makes it break we can almost certainly fix it. I haven't managed to get it to happen myself yet.
